JS-do-not-construct-regex-from-userinput/mohandeepak99_Hotel_Reservation_app.js

```
@@ -0,0 +1,274 @@
+});
+app.get("/search/:searchtext?",(req,res)=>{
+
+    console.log(req.params);
+
+    if(req.params.searchtext!=undefined){
+
+        var search = new RegExp(req.params.searchtext, 'i');
+        var searchcond = {uname :search};
+    }
+    else{
+        var searchcond = null;    
```

Biome (2.1.2)

[error] Shouldn't redeclare 'searchcond'. Consider to delete it or rename it.

'searchcond' is defined here:


(lint/suspicious/noRedeclare)